flight/mcu/lib/teensyShared/rocketFSM.h

```diff
@@ -12,7 +12,7 @@ struct pointers;
 class RocketFSM {
    public:
     /**
-     * @brief Labels for each FSM state
+     * @brief Labels for each FSM state. Contains intermediary states (eg: Launch Detect) along with actual states (eg: Boost)
      */
     enum class FSM_State {
```

Comment:
I think this enum should probably be extracted to a different location, to completely separate the values that are logged from the code that uses said values.